backend/config/config_template.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
```

Comment:
Make sure to update the config loader to validate the config with the pydantic model and replace the config variable with the pydantic object. Make sure that all references to the config are updated accordingly.